tests/docker/readme.md

```diff
@@ -0,0 +1,96 @@
+Example of docker container initialization log, how `php app migrate/wait` in `entrypoint.sh` affects the startup.
```

Comment:
Please clarify why someone might need these logs. 👍

Reply:
It lasted a while I understood, what goes on when the php and db container is starting paralel, and php container sometimes can, sometimes can't connect to the database. This is the case when php container is really have to wait for the database before can connect and run the migrations. The log demonstrates the need of migration/wait command.

Can you explain it better? 
I don't insist on it, it can even be deleted.

Reply:
OK, I see. Maybe the following sentences can be added at the end of the one already there.

- It illustrates several trial/failure attempts to connect to the database until success, so that migrations can be run if any. This explains why `migrate/wait` should be used with docker to wait for database container initialization before running migrations.